lms/static/js/groups/views/cohort_discussions_course_wide.js

```diff
@@ -15,7 +15,7 @@
 
                     render: function () {
                         this.$('.cohort-course-wide-discussions-nav').html(this.template({
-                            courseWideTopics: this.getCourseWideDiscussionsHtml(
+                            courseWideTopicsHtml: this.getCourseWideDiscussionsHtml(
```

Comment:
Note to reviewers: renaming these variables that return HTML for clarity in the template.